packages/ckeditor5-package-generator/lib/utils/validate-global-name.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license Copyright (c) 2020-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+
+/**
+ * If the global name is not valid, prints the error and exits the process.
+ *
+ * @param {Logger} logger
+ * @param {String|undefined} globalName
```

Comment:
The `globalName` is always defined.

```suggestion
 * @param {String} globalName
```